### ISSUE TITLE
Strange interaction between -h option and beam angles

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#232]
Here is the standard output of verovio for the file
https://raw.githubusercontent.com/rism-ch/verovio/gh-pages/examples/downloads/Chopin_Etude_op.10_no.9.mei

``` bash
./verovio -f mei Chopin_Etude_op.10_no.9.mei
```

Rendering of the first system:

![screen shot 2016-07-04 at 9 35 57 pm](https://cloud.githubusercontent.com/assets/3487289/16574023/56987c04-422f-11e6-97e8-0f652dc13238.png)

Now adding `-h 20000` to the command-line:

``` bash
./verovio -f mei Chopin_Etude_op.10_no.9.mei -h 20000
```

Produces this first system:

![screen shot 2016-07-04 at 9 37 22 pm](https://cloud.githubusercontent.com/assets/3487289/16574041/93c74182-422f-11e6-8aba-ef2f2bf63d26.png)

Notice that the beams have changed, only due to the addition of `-h 20000`.  So something strange is going on.  Mostly it seems that beams on the right half of the page will change.  And it looks like their stating points are the same, but the slope is negated.

**Comment by k-ljung** [comment]
I have a simular issue:
```
verovio -f musicxml "Back in the USSR-WaV-124bpm.xml"
```
or
```
verovio -f musicxml -h 2000 "Back in the USSR-WaV-124bpm.xml"
```
Produces:
![screen shot 2017-06-29 at 22 40 50](https://user-images.githubusercontent.com/9248053/27709374-197f5bea-5d1c-11e7-9633-1fb796bbc8d0.png)
This seems to be okay, but:

```
verovio -f musicxml -h 15000 "Back in the USSR-WaV-124bpm.xml"
```
Produces:

![screen shot 2017-06-29 at 22 44 30](https://user-images.githubusercontent.com/9248053/27709486-9025282e-5d1c-11e7-93dd-472792b1dd37.png)

```
verovio -f musicxml -h 20000 "Back in the USSR-WaV-124bpm.xml"
```
Produces:
![screen shot 2017-06-29 at 22 47 24](https://user-images.githubusercontent.com/9248053/27709594-fbbb4da2-5d1c-11e7-8c6c-ede2332f27bf.png)

```
verovio -f musicxml -h 30000 "Back in the USSR-WaV-124bpm.xml"
```
Produces:
![screen shot 2017-06-29 at 22 49 03](https://user-images.githubusercontent.com/9248053/27709650-2e15e0dc-5d1d-11e7-98f8-31d1391063ea.png)

```
verovio -f musicxml -h 60000 "Back in the USSR-WaV-124bpm.xml"
```
Produces:
![screen shot 2017-06-29 at 22 50 35](https://user-images.githubusercontent.com/9248053/27709724-7220358e-5d1d-11e7-8b97-720d7f73d6a4.png)

**Comment by lpugin** [comment]
Yes, this is pretty clearly and int overflow in the beam slope calculation.

**Comment by lpugin** [comment]
Fix f6a21feab2a590e1199ad9e870bc8298301ae27f
